pkg/operator/resource/resourceapply/unstructured.go

```diff
@@ -0,0 +1,19 @@
+package resourceapply
+
+import (
+	"fmt"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic"
+)
+
+// ApplyUnstructured applies few selected Unstructured types. Feel free to add more.
+func ApplyUnstructured(client dynamic.Interface, recorder events.Recorder, obj *unstructured.Unstructured) (*unstructured.Unstructured, bool, error) {
+	switch obj.GetKind() {
+	case "ServiceMonitor":
```

Comment:
need to use the full GVK

Reply:
Well, I guess GroupKind is probably sufficient.

Reply:
Added full gvk.